chaos-monkey-spring-boot/src/main/java/de/codecentric/spring/boot/chaos/monkey/endpoints/ChaosMonkeyRestEndpoint.java

```diff
@@ -42,6 +42,10 @@
 
   private final ChaosMonkeyScheduler scheduler;
 
+  private ChaosMonkeyEnabledDto chaosMonkeyEnabledDto;
+
+  private ChaosMonkeyDisabledDto chaosMonkeyDisabledDto;
```

Comment:
I think its better to store the state when chaos monkey was enabled in chaosmonkeysettings/chaosmonkeyproperties instead of here. Then you dont need to duplicate to Rest and JmxEndpoint